prompt_tuning/utils.py

```diff
@@ -20,15 +26,102 @@ def seed_everything(seed):
 
 
 def print_result(
-    test_type: str, step: int, loss: float, accuracy_score: float, f1_score: float, epoch: Optional[int] = None
+    test_type: str,
+    step: int,
+    loss: float,
+    accuracy_score: Optional[float] = None,
+    f1_score: Optional[float] = None,
+    epoch: Optional[int] = None,
 ) -> None:
     if step != 0:
-        log.info(
-            (
-                f"[{test_type}] "
-                f"Epoch {epoch if epoch is not None else ''} "
-                f"{test_type} loss: {loss / step} "
-                f"accuracy: {accuracy_score / step} "
-                f"f1_score: {f1_score / step}"
+        if test_type == "train":
+            log_string = f"[{test_type}] Epoch {epoch} loss: {loss / step} "
+            log.info(log_string)
```

Comment:
```suggestion
            log.info(f"[{test_type}] Epoch {epoch} loss: {loss / step} ")
```

굳이 다시 사용되는 변수가 아니라면, 리터럴로 써도 될 거 같아요.

Reply:
else 이하도 마찬가지라고 생각하는데, else이하는 string이 길어져서 뺀거면 OK!

Reply:
적용하겠습니다~!